scripts/west-win.py

```diff
@@ -0,0 +1,11 @@
+# Windows-specific launcher alias for west (west wind?).
```

Comment:
I was really looking forward to seeing a "west.bat", as a pun with Adam West.

Reply:
Oh, I would love that if you know how to write one :+1:. I took the approach I did mostly to avoid having to learn more about .bat scripts...